spec/factories/locations.rb

```diff
@@ -1,7 +1,16 @@
 FactoryBot.define do
   factory :location do
-    name { "MyString" }
+    sequence(:name) { |n| "Location #{n}" }
     facility
     organization
+
+    trait :with_enclosures do
+      transient { enclosures_count { 5 } }
+
+      after(:create) do |loc, eval|
+        create_list :enclosure, eval.enclosures_count, location: loc
+        loc.reload
+      end
```

Comment:
Its more verbose, but I think i prefer the full `location` here for clarity.

Reply:
I think I'll have a few changes in #445 then 😆